### PR TITLE
Clean up requirements.txt

### PR DESCRIPTION
**Pull request by jeylau**
- [x] Get rid of moviepy (which prevents users from having numpy >= 1.20, also causing errors with TF at times)
- [x] Remove jupyter-books (it's already being installed automatically when publishing our docs, and this will let users install ipython >= 8, which has a very nice (predictive) autocomplete feature)
- [x] Ensure scikit-learn >= 1.0
- [x] Pin tables to their newest release (notably with support for numpy 1.21 and HDF5 1.12)

And no ugly red crosses anymore!

### CODE REVIEW

**Review by aristotelis-economides — APPROVED** [review]
i like removing the need to update things in two places :muscle: :rocket:

File: setup.py [hunk 1]
```diff
@@ -10,10 +10,20 @@
 """
 
 import setuptools
+from pathlib import Path
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+requirements = []
```
Comment (MMathisLab) [inline setup.py]: is this required, or why the addition? @jeylau

**Review by MMathisLab — CHANGES_REQUESTED** [review]
left one comment, then great!

File: setup.py (hunk 1 above)
Comment (jeylau) [reply setup.py]: My bad, I had undone the commit but this left this part. I'll remove it :)

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**